### ISSUE TITLE
Could our paper be added?

### ISSUE THREAD

**Issue by johntzwei** [lyy1994/awesome-data-contamination#10]
Hi, thank you for maintaining this great repo! I wanted to ask if our paper could be added:

Spiking the training data to correct for test set contamination
Paper: https://arxiv.org/abs/2605.24818
Code: https://github.com/Jeli04/spiking-tsc

This work studies the correction of test set contamination, using the Hubble models (a minimal pair of 8B LLMs, where one was trained normally, and the other was trained in the same way but with some test set contamination). We provide some statistical perspectives on this problem, and propose a few estimators to correct inflated test scores. To evaluate estimators, they are given the contaminated model and their correction target is the clean LLM's test score.

Thank you for your consideration!

**Comment by lyy1994** [comment]
Your paper has been added. Thanks for letting us know about your work!
